demos/demos_by_use_case/bio/cpu/metagenomic_before_gpu.ipynb

```diff
@@ -0,0 +1,721 @@
+        "\n",
+        "**Insight/ Result:**\n",
+        "\n",
+        "2342s to umap and dbscan, which [the analysis reference paper](https://journals.asm.org/doi/full/10.1128/msystems.00118-23) does 12x per dataset\n",
+        "\n",
+        "(See also: [GPU acceleration via graphistry](https://github.com/dcolinmorgan/grph/blob/main/accelearting_metagenomic_demo.ipynb))"
+      ],
+      "metadata": {
+        "id": "tLRHg2VEzoYy"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PjnS_PCWaClg"
+      },
+      "source": [
+        "# Setup"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "JTzHVRgEcDUV"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install graphistry[ai]\n",
+        "!pip install dirty_cat umap-learn\n",
+        "import umap\n",
+        "import dirty_cat\n",
+        "!pip install Biopython"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# import /configure\n",
+        "\n",
+        "get a free api-key at https://www.graphistry.com/\n"
+      ],
+      "metadata": {
+        "id": "HK_8_7UB0mhx"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import pandas as pd\n",
+        "import graphistry\n",
+        "from time import time\n",
+        "\n",
+        "\n",
+        "graphistry.register(api=3,protocol=\"https\", server=\"hub.graphistry.com\", username='dcolinmorgan', password='fXjJnkE3Gik6BWy') ## key id, secret key\n",
+        "\n",
+        "# graphistry.register(api=3,protocol=\"https\", server=\"hub.graphistry.com\", username='dcolinmorgan', password='***') ## key id, secret key\n",
+        "graphistry.__version__"
+      ],
+      "metadata": {
+        "id": "i42QrQ_ejC4h",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 35
+        },
+        "outputId": "58e4c0eb-11e1-448f-896b-3931664a09da"
+      },
+      "execution_count": 4,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "'0.29.1'"
+            ],
+            "application/vnd.google.colaboratory.intrinsic+json": {
+              "type": "string"
+            }
+          },
+          "metadata": {},
+          "execution_count": 4
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# bio-ml dataset\n",
+        "\n",
+        "\n",
+        "1.   [3 subjects x 10 time points](\n",
+        "https://www.ebi.ac.uk/ena/browser/view/PRJNA544527)\n",
+        "\n",
+        "2.  [metadata](\n",
+        "https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-019-0559-3/MediaObjects/41591_2019_559_MOESM3_ESM.xlsx)\n",
+        "\n",
+        "3.   !wget https://raw.githubusercontent.com/dcolinmorgan/grph/main/ftp_PRJNA544527.txt\n"
```

Comment:
should we move this into the graphistry githubusercontent or as a txt in this demo folder?